### ISSUE TITLE
F841 local variable is assigned but never used

### ISSUE THREAD

**Issue by busunkim96** [googleapis/gapic-generator-python#425]
```
tests/unit/texttospeech_v1beta1/test_text_to_speech.py:199:9: F841 local variable 'response' is assigned to but never used
tests/unit/texttospeech_v1beta1/test_text_to_speech.py:262:9: F841 local variable 'response' is assigned to but never used
tests/unit/texttospeech_v1beta1/test_text_to_speech.py:305:9: F841 local variable 'client' is assigned to but never used
tests/unit/texttospeech_v1/test_text_to_speech.py:154:9: F841 local variable 'client' is assigned to but never used
tests/unit/texttospeech_v1/test_text_to_speech.py:199:9: F841 local variable 'response' is assigned to but never used
tests/unit/texttospeech_v1/test_text_to_speech.py:262:9: F841 local variable 'response' is assigned to but never used
tests/unit/texttospeech_v1/test_text_to_speech.py:305:9: F841 local variable 'client' is assigned to but never used
```

https://github.com/googleapis/python-texttospeech/pull/30

**Comment by miraleung** [comment]
@software-dov PTAL?

**Comment by software-dov** [comment]
Fixed by #428
